### DISCUSSION TITLE
OSM into Cityjson

### DISCUSSION THREAD

**Discussion by kaheetonaa** [cityjson/cjio#149] [Ideas]
I am thinking about turning OSM data into Cityjson data by using cjio, then editing by Blender by Up3date (reference for detail from crowdsourcing Street View Imageries or Public Domain image or self obtain). Then we can store each object 3d semantic model on database like Mongodb. As a result, a OSM object can have a 3d cityjson model (which can also be linked by distinctive tag). Is this possible to create such framework?

**Comment by balazsdukai** [comment]
Yes, it is probably possible. There was some work on storing CityJSON data in Mongodb, maybe that can give you some pointers https://repository.tudelft.nl/islandora/object/uuid%3A7f9209f7-248f-4c93-9ba3-5866655e6040.

Do you mean that you want to texture the models in Blender? Or what is the need for editing them in Blender?

**Reply by kaheetonaa** [reply]
In Blender user can add or edit architectonic object (roof column window door) so that the model would be more detailed. In the case of building, after being extruded to the estimated height, we got the LoD0, then import them in Blender to add roof or other detail (texture is one of them) based on crowdsourced image (if possible) we will draftly have LoD1 (correct me if anything false).

**Reply by balazsdukai** [reply]
Yes it's clear now, thanks. It should be feasible what you described, although never done it myself.
